oscqam/actions/rejectaction.py

```diff
@@ -15,12 +15,15 @@ class RejectAction(OscAction):
 
     DECLINE_MSG = "Declining request {request} for {user}. See Testreport: {url}"
 
-    def __init__(self, remote, user, request_id, reason, message=None, out=sys.stdout):
+    def __init__(
+        self, remote, user, request_id, reason, force, message=None, out=sys.stdout
+    ):
         super(RejectAction, self).__init__(remote, user, out=out)
         self.request = remote.requests.by_id(request_id)
-        self._template = None
+        self._template = None if not force else "There isn't template"
```

Comment:
```suggestion
        self._template = None if not force else "There is no template"
```
(because "isn't" usually means "is not")